runtime/doc/treesitter.txt

```diff
@@ -818,6 +818,10 @@ get_parser({bufnr}, {lang}, {opts})              *vim.treesitter.get_parser()*
 
     If needed, this will create the parser.
 
+    If no parser can be created, an error is thrown. Set `opts.error = false`
+    to suppress this and return nil instead. WARNING: This behavior will
+    become default in Nvim 0.12 and the option will be removed.
```

Comment:
The option becoming default and the option being removed should happen in separate releases. Might want to reword here to clarify that. “This behavior will become default in Nvim 0.12 and removed in a future release”

Reply:
No, the proposal is in fact for an expedited deprecation (flying under the "experimental" flag). If there are _specific_ objections to that (and the related API change; this is a bigger project), I would like to hear them. (This change is _much_ easier to adapt to than the query iteration one.)

Reply:
To be more precise: What I'm looking for here is scenarios where the option removal will be breaking whereas the default switch would not be. (Note that you're opting in to the new behavior, not the old one, from the start -- unlike the query change.)